bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+# TODO
+# We are using external apps just for the moment.
+# These will be removed for native libraries.
+AWSCLIENT = "/usr/local/bin/aws"
+PGCLIENT = "/usr/bin/psql"
+
+# NZSL: Is there a database url defined in the environment?
+DATABASE_URL = os.getenv("DATABASE_URL", None)
+
+parser = argparse.ArgumentParser(
+    description="You must have a configured AWSCLIENT profile to use this app. See the --awsprofile "
+    "argument."
+)
+# Positional arguments
+if DATABASE_URL:
+    print("DATABASE_URL defined in environment", file=sys.stderr)
+else:
+    parser.add_argument(
+        "dburl",
+        help=f"(REQUIRED) Database url (Overridden by DATABASE_URL environment variable)",
+    )
+# Optional arguments
+parser.add_argument(
+    "--awsprofile",
+    default="nzsl",
+    required=False,
+    help=f"AWS configured profile to use (default: '%(default)s')",
+)
+parser.add_argument(
+    "--production",
+    default=False,
+    required=False,
+    action="store_true",
+    help="Run in PRODUCTION mode, instead of STAGING (default: %(default)s)",
+)
+parser.add_argument(
+    "--cached",
+    default=False,
+    required=False,
+    action="store_true",
+    help="Use keys generated on a previous non-cached run (default: %(default)s) "
+    "(Don't mix PRODUCTION and STAGING!)",
+)
+parser.add_argument(
+    "--pgclient",
+    default=PGCLIENT,
+    required=False,
+    help=f"Postgres client path (default: %(default)s)",
+)
+parser.add_argument(
+    "--awsclient",
+    default=AWSCLIENT,
+    required=False,
+    help=f"AWS client path (default: %(default)s)",
+)
+args = parser.parse_args()
+
+if args.production:
+    MODE_STR = "PRODUCTION"
+    NZSL_APP = "nzsl-signbank-production"
+    AWS_S3_BUCKET = "nzsl-signbank-media-production"
+else:
+    MODE_STR = "STAGING"
+    NZSL_APP = "nzsl-signbank-uat"
+    AWS_S3_BUCKET = "nzsl-signbank-media-uat"
```

Comment:
issue: I think it would be cleaner if we took an environment name and then inferred these from that

we're only using `MODE_STR` and `args.production` in one place (the former is printed as part of an output, and the latter is used here), and the values here are consistent - so if we switched to instead taking the values `production` and `uat`, we could simplify this by just prefixing that value with `nzsl-signbank-` and `nzsl-signbank-media-`.

Reply:
Done.